src/ontology/edits/oeo-shared.omn

```diff
@@ -1925,16 +1925,19 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1078"@en,
 Class: OEO_00240024
 
     Annotations: 
-        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy service demand is a disposition of an agent to use energy as a mean to obtain or facilitate desired end services or states."@en,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "An energy service demand is a disposition of an agent to use energy as a means to obtain or facilitate desired end services or states."@en,
         <http://purl.obolibrary.org/obo/IAO_0000116> "An example is the transport demand for goods or persons (e.g. in person/tonne-kilometres). An other example is the demand for room heating or cooling."@en,
         <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1054
 pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1080",
         rdfs:label "energy service demand"
     
+    EquivalentTo: 
+        OEO_00010121 some (<http://purl.obolibrary.org/obo/RO_0000087> some OEO_00000051)
```

Comment:
I think it should be a subclass of axiom rather than an equivalent to axiom, as this relationship to the bearer does not *fully define* the entity, rather, it just adds more information about it.

Reply:
Thank you! This was the reason!